core-bundle/src/Resources/contao/library/Contao/Database/Statement.php

```diff
@@ -228,16 +220,7 @@ public function limit($intRows, $intOffset=0)
 	 */
 	public function execute()
 	{
-		$arrParams = \func_get_args();
-
-		if (\count($arrParams) === 1 && \is_array($arrParams[0]))
-		{
-			trigger_deprecation('contao/core-bundle', '4.13', 'Using "%s()" with an array parameter has been deprecated and will no longer work in Contao 5.0. Use argument unpacking via ... instead."', __METHOD__);
```

Comment:
Did you check that we don‘t accidentally do this?

Reply:
I expect us not to if it's deprecated.